src/interface.ts

```diff
@@ -1517,6 +1517,10 @@ export interface EventCfg {
  */
 export type SliderOption = SliderCfg | boolean;
 
+export interface MouseWheelScrollCfg {
+  wheelSpeed?: number
```

Comment:
Using constant `wheelSpeed` causes scrolling information to be lost, consider using `wheelSpeedRatio` instead.

Reply:
The logic behind the `wheelSpeed` is described in the comment above. On each scroll event the next displayed chart's data item index is calculated. The next ratio is then calculated to guarantee that the scrollbar component will properly filter the chart's data and the data item with this index will be the first data item to be displayed after scroll.

Reply:
I initially assumed that the chart's data items scroll at the same speed as the scroll wheel, that is, the faster the scroll wheel moves, the faster the chart items switch. If constant `wheelSpeed` is specified, the actual wheel speed feedback from the mouse is ignored.